ui/desktop/app/styles/app.scss

```diff
@@ -652,3 +652,13 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     border-radius: 0 0 5px 5px;
   }
 }
+
+#terminal-container {
+  .terminal.xterm .xterm-viewport {
+    border-radius: 8px;
+  }
+
+  .xterm-rows {
+    padding: 0.5rem;
```

Comment:
Should this just be padding-left?

Reply:
I don't think you can just add straight `padding` to the top or bottom since it will cause the terminal to overflow at the bottom

<img width="1840" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/ceb59efb-66b3-4bea-8dc1-683577824c73">

<img width="1392" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/78e8d4f8-3485-4dc4-943a-db7110d1b292">

Reply:
Look at me forgetting all of the style headaches from session recording. Thanks for checking that. I will update now.